R/add_company_target.R

```diff
@@ -19,7 +19,9 @@
 #' master <- r2dii.data::loanbook_demo %>%
 #'   r2dii.match::match_name(r2dii.data::ald_demo) %>%
 #'   r2dii.match::prioritize() %>%
-#'   join_ald_scenario(r2dii.data::ald_demo, r2dii.data::scenario_demo_2020)
+#'   join_ald_scenario(r2dii.data::ald_demo,
+#'                     r2dii.data::scenario_demo_2020,
+#'                     region_isos = region_isos_demo)
```

Comment:
While 0-row output is a valid option, it would be most likely a mistake. Should we add a warning? (Or at lease a silent, generic signal that we can catch and test for so we don't get this bug again.)

Reply:
what is the best way to test @examples?

Reply:
Interactively I use `devtools::run_examples(fresh = TRUE)`
While `test()`ing, maybe just copy-paste into a formal test.